client/state/comments/README.md

```diff
@@ -0,0 +1,29 @@
+Comments
+========
+
+## This store has the following structure:
+
+```
+comments = {
+	items: Map<CommentTargetId, Tree>,
```

Comment:
I think I'd rather nest all of the info under one Map of CommentTargetId to named bits. Having all these maps that take the CommentTargetId as the key makes it harder to just drop a set of comments and their info.